content/en/docs/guides/traffic_management/multi_cluster.md

```diff
@@ -0,0 +1,70 @@
+
+In this part of the series, we will be looking into motivation, goals, architecture of [FSM](https://github.com/flomesh-io/fsm) multi-cluster support, its components, and how it integrates with [osm-edge](https://flomesh.io/osm-edge) full-fledged, lightweight, SMI-compatible service mesh suitable for cloud-native and edge.
+
+## Flomesh Service Mesh
+
+[FSM](https://github.com/flomesh-io/fsm) is an open-source product from [Flomesh](https://flomesh.io) for Kubernetes north-south traffic, gateway API controller, and multi-cluster management. FSM uses [Pipy](https://flomesh.io), a programmable proxy at its core, and provides an Ingress controller, *Gateway API* controller, load balancer, cross-cluster service registration discovery, and more.
+
+### Motivation
+
+During our consultancy and support to the community, commercial clients, and enterprises we have seen multiple requests and desires (a few of which are cited above) on why they want to split their deployments across multiple clusters while maintaining mutual dependencies between workloads operating in those clusters. Currently, the cluster is a hard boundary, and service is opaque to a distant K8s consumer who may otherwise use metadata (e.g. endpoint topology) to better direct traffic. Users may want to use services distributed across clusters to support failover or temporarily during migration, however, this needs non-trivial customized solutions today.
+
+Flomesh team aims to help the community by providing solutions to these problems.
+
+### Goals
+
+* Define a minimal API to support service discovery and consumption across clusters.
+  * Consume a service in another cluster.
+  * Consume a service deployed in multiple clusters as a single service.
+* When a service is consumed from another cluster its behavior should be predictable and consistent with how it would be consumed within its cluster.
+* Allow gradual rollout of changes in a multi-cluster environment.
+* Provide a stand-alone implementation that can be used without any coupling to any product and/or solution.
+* Transparent integration with [osm-edge](https://flomesh.io/osm-edge) service mesh, for users who want to have multi-cluster support with service mesh functionality.
+* Fully open source and welcomes the community to participate and contribute.
+
+### Architecture
+
+* Control plane
+
+ ![](/docs/images/mcs/fsm-architecture-cp.png)
+
+* osm-edge integration (managed cluster)
+
+ ![](/docs/images/mcs/fsm-architecture-managed-cluster.png)
+
+FSM provides a set of Kubernetes custom resources (CRD) for cluster connector, and make use of [KEP-1645](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api) `ServiceExport` and `ServiceImport` API for exporting and importing services.
+
```

Comment:
I think it's better to add an introduction to all CRDs related to MCS here.

Reply:
we can give a simple sample of CRD to introduce the feature that CRD provides. Such as ServiceImport, ServiceExport, and GlobalTraficPolicy.

The reader should know what CRD introduced in MCS and its main feature. For details, refer to the demo.